### ISSUE TITLE
Classic Era Update 1.15.1

### ISSUE THREAD

**Issue by maximus210793** [Pazza/ReagentCount#2]
Hi,
first of all, I want to say great adddon!
Could you update it to the latest Classic Era Game  Version of 1.15.1?
Would be awesome, thanks a lot in advance!

**Comment by Pazza** [comment]
Done

**Comment by maximus210793** [comment]
Hi, when can I download the update via Curseforge App?
Currently still last relase 10/06/2023.
Thanks in advance for already updating :)

**Comment by Pazza** [comment]
Let me double check. I thought it was set up to auto package.

**Comment by maximus210793** [comment]
Update: Can still not update to latest version.

**Comment by Pazza** [comment]
Yeah, I'm not sure why the packager is not working. Feel free to grab the ReagentCount.toc file from here and replace yours in the addon folder in the meantime

**Comment by maximus210793** [comment]
Thanks for the tip, I did so.
But still on my paladin on every button it shows the "0" for example for greater Blessing of Might etc., but I have 950 Symbols of Kings in my invenatory, do you know why it is not displaying the right amount ?

**Comment by Pazza** [comment]
Shoot, I do not. I thought you just needed me to bump the TOC version. I'd have to take a look into it. I don't play anymore, but still keep an account. You'll have to be patient while I find time to look.

**Comment by maximus210793** [comment]
Ok sorry for the misunderstanding.
Would be nice, if you could fix it, so useful your addon <3
Thanks in advance!

**Comment by maximus210793** [comment]
For paladin it is working again, but not for my shaman with e.g.:  Water Breathing &  Water Walking

**Comment by maximus210793** [comment]
Could you reopen this ticket, since I saw it is closed?

**Comment by maximus210793** [comment]
Hi mate,
any update on this issue?
Still not working with latest WoW Classic Era Release 1.15.2.
Thanks in advance!

**Comment by Pazza** [comment]
No not yet, I'm crossing a milestone at work. I hope to have some time in the next week or two
